src/gpio/stm32l496ag.rs

```diff
@@ -0,0 +1,65 @@
+//! GPIO specific to STM32L496AG
+
+use ::stm32l4x6;
+
+use super::*;
+
+impl_parts!(
+    GPIOD, gpioc;
+    GPIOE, gpioc;
+    GPIOF, gpioc;
+    GPIOG, gpioc;
+    GPIOH, gpioc;
+);
+
+/// Description of GPIOs and PINs
+pub mod gpio {
+    pub use super::*;
+
+    impl_gpio!(D, GPIOD, gpioden, gpiodrst,
+               AFRL: [PD0, 0; PD1, 1; PD2, 2; PD3, 3; PD4, 4; PD5, 5; PD6, 6; PD7, 7;],
+               AFRH: [PD8, 8; PD9, 9; PD10, 10; PD11, 11; PD12, 12; PD13, 13; PD14, 14; PD15, 15; ]
+    );
+    impl_gpio!(E, GPIOE, gpioeen, gpioerst,
+               AFRL: [PE0, 0; PE1, 1; PE2, 2; PE3, 3; PE4, 4; PE5, 5; PE6, 6; PE7, 7;],
+               AFRH: [PE8, 8; PE9, 9; PE10, 10; PE11, 11; PE12, 12; PE13, 13; PE14, 14; PE15, 15; ]
+    );
+    impl_gpio!(F, GPIOF, gpiofen, gpiofrst,
+               AFRL: [PF0, 0; PF1, 1; PF2, 2; PF3, 3; PF4, 4; PF5, 5; PF6, 6; PF7, 7;],
+               AFRH: [PF8, 8; PF9, 9; PF10, 10; PF11, 11; PF12, 12; PF13, 13; PF14, 14; PF15, 15; ]
+    );
+    impl_gpio!(G, GPIOG,  gpiogen, gpiogrst,
+               AFRL: [PG0, 0; PG1, 1; PG2, 2; PG3, 3; PG4, 4; PG5, 5; PG6, 6; PG7, 7;],
+               AFRH: [PG8, 8; PG9, 9; PG10, 10; PG11, 11; PG12, 12; PG13, 13; PG14, 14; PG15, 15; ]
+    );
+    impl_gpio!(H, GPIOH, gpiohen, gpiohrst,
+               AFRL: [PH0, 0; PH1, 1; PH2, 2; PH3, 3; PH4, 4; PH5, 5; PH6, 6; PH7, 7;],
```

Comment:
Can you please check PH3 for whether it can have alternative functions?
As you saw there was comment in code about it being special but I don't remember right now details.

You might want to check if on your board it is possible to configure alternative functions on this pin?